bundle/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/TCS.scala

```diff
@@ -136,15 +126,13 @@ object TCS {
     }
   }
 
-  def fromSequenceConfig(config: Config)(s0: TcsConfig): TcsConfig = {
+  def fromSequenceConfig(config: Config)(s0: TcsConfig): TcsConfig =
     List(
       build(buildPwfs1Config,  TELESCOPE_KEY / GUIDE_WITH_PWFS1_PROP, config),
       build(buildPwfs2Config,  TELESCOPE_KEY / GUIDE_WITH_PWFS2_PROP, config),
       build(buildOiwfsConfig,  TELESCOPE_KEY / GUIDE_WITH_OIWFS_PROP, config),
       build(buildOffsetConfig, TELESCOPE_KEY / P_OFFSET_PROP, TELESCOPE_KEY / Q_OFFSET_PROP, config)
-    ).foldLeft(s0)((b, f) => f(b))
-
-  }
+    ).suml.apply(s0)
```

Comment:
Does this takes all the `Endo`s and `|+|` them together, and then apply `A=>A` to `s0`?